package.json

```diff
@@ -5,6 +5,7 @@
   "author": "JSConf Hawaii <hello@jsconfhi.com>",
   "dependencies": {
     "babel-plugin-styled-components": "^1.10.6",
+    "eslint-plugin-flowtype": "^4.3.0",
```

Comment:
Remove?

Reply:
Gatsby on Windows didn't want to work with me without it (maybe I'm missing something), but I can remove it.

Reply:
What error were you seeing? On which command?

Reply:
I get the following error:
```Module build failed (from ./node_modules/eslint-loader/index.js):
Error: Failed to load plugin flowtype: Cannot find module 'eslint-plugin-flowtype'```

Followed by

```ERROR in ./.cache/app.js
Module build failed (from ./node_modules/eslint-loader/index.js):
Error: Failed to load plugin flowtype: Cannot find module 'eslint-plugin-flowtype'```

Reply:
For now, I've removed the dependency from the package.json and am just keeping the plugin in my local environment, which allows me to start the gatsby dev server. Happy medium.